assembly/guide-configs/app-dev/java/foundations.json

```diff
@@ -47,6 +47,10 @@
       "type": "h2",
       "id": "java/how-to-develop-a-workflow-definition-in-java"
     },
+    {
+      "type": "h3",
+      "id": "java/interface-heirarchies"
```

Comment:
Probably a legacy issue, but I noticed that "hierarchies" is misspelled here (and in the name of the file itself as well as other references to it).

Reply:
No, that's me :) Not sure what I was thinking there.